FrontEnd/src/components/AboutPage/MeetTheTeam/MemberInfoCard/MemberInfoCard.tsx

```diff
@@ -1,7 +1,108 @@
 import React, { FunctionComponent } from 'react';
 import { Col, Container, Row } from 'react-bootstrap';
 import styles from './MemberInfoCard.module.scss';
+import Card from 'react-bootstrap/Card';
+import { contactIcons } from '@icons';
+import { Link, FilledImage } from '@basics';
+import { Body1, Body2, Caption } from '@components/basics';
+import cn from 'classnames';
 
-const MemberInfoCard: FunctionComponent = () => <Container></Container>;
+export interface MemberInfoCardProps {
+  name: string;
+  imgSrc: string;
+  title: string;
+  quote: string | { number: number; length: string };
+  website?: string;
+  email?: string;
+  linkedIn?: string;
+}
+
+const MemberInfoCard: FunctionComponent<MemberInfoCardProps> = ({
+  name,
+  imgSrc,
+  title,
+  quote,
+  website,
+  email,
+  linkedIn,
+}) => {
+  //establishing a first name for use in text bodeis
+  var nameArray: string[] = name.split(' ');
+  var firstName: string = nameArray[0];
+  //changing the grammar for design team
+  var team: string = title.toLowerCase();
+  team === 'designer' ? (team = 'design') : null;
+
+  //left side of the card, consists of picture, name, title, and quote
+  const infoCol = (
+    <div className={styles.firstCol}>
+      <div className={styles.information}>
+        <Row>
+          <FilledImage src={imgSrc} alt="Image" className={styles.picture} />
+
+          <Col className={styles.nameinfo}>
+            <div className={styles.name}>
+              <Body1>{name}</Body1>
+            </div>
+
+            <div className={styles.title}>
+              <Caption>{title}</Caption>
+            </div>
+          </Col>
+        </Row>
+        <div className={styles.caption}>
+          {typeof quote === 'string' ? (
+            <Body2>"{quote}"</Body2>
+          ) : (
+            <Body2>
+              {firstName} has been on the {team} team for {quote.number}{' '}
+              {quote.length} long
+            </Body2>
+          )}
+        </div>
+      </div>
+    </div>
+  );
+
+  const teamColor =
+    team.toLowerCase() === 'design'
```

Comment:
team.toLowerCase() is reused a ton in this. Consider if this can be removed and put into a different variable instead.

i.e. const t = team.toLowerCase();
const teamColor = t === 'design' ? styles.design.... etc

Also, there's a cleaner way to do this. Checkout how to utilize classnames here and focus on how to express conditional classes as an object: https://github.com/JedWatson/classnames#usage-with-reactjs
Reach out if you need clarification